README.md

```diff
@@ -53,34 +53,39 @@ which is supported by the tldr-pages project maintainers:
 
     npm install -g tldr
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+Alternatively, you can also use the fully-featured official [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
 
     pip3 install tldr
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+Then you have direct access to less complicated, easier-to-read help for commands, such as tar,
+accessible through typing `tldr tar` instead of the standard `man tar`
 
 However, if you just want to browse without installing anything, check
-out the [PDF version](https://tldr.sh/assets/tldr-book.pdf).
+out the [PDF version](https://tldr.sh/assets/tldr-book.pdf),
+which is always up-to-date with the most recent pages and commits.
 
 There are also **various other clients** provided by the community,
 both for the command-line and for other platforms.
 For a comprehensive list of clients, head over to our [Wiki](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients).
 
-## How do I contribute?
+## How do I contribute to tldr-pages?
 
-- Your favourite command isn't covered?
-- You can think of more examples for an existing command?
+All contributions are welcome!
+
+Some ways to contribute include:
+
+- Adding your favorite a command which isn't covered.
+- Adding more to an existing page.
+- Translating pages into different languages.
 
 All `tldr` pages are kept as Markdown files right here in this repository,
 so you can edit them directly and submit your changes as pull requests.
 
-All contributions are welcome!
 We strive to maintain a [welcoming and collaborative](GOVERNANCE.md) community.
-Have a look at the [contributing guidelines](CONTRIBUTING.md), and go ahead!
+To begin, have a look at the [contributing guidelines](CONTRIBUTING.md), and go ahead!
 
-If you'd like to contribute to translations, you can visit <https://lukwebsforge.github.io/tldri18n/>
-to see the current progress of all translations.
+For translations, you can visit <https://lukwebsforge.github.io/tldri18n/>
```

Comment:
I prefer the original wording. Why are you proposing this change?

Reply:
Are you referring to line 88 as well?

I assume only 87.

From the changes made above, I think it fits better with the changed content specifically new line 79 where translations as a possible option has been listed, so the subject has already been stated and now this paragraph just continues where it was left off.

Reply:
I lost track of the line numbers now that you've made changes that affected them :sweat_smile: but IIRC, I was referring to the change "If you'd like to contribute to translations" → "For translations".

And yeah, I get your point, but I still prefer the sentence to be explicit and not depend on the reader reading the content of this file in full and in sequence.

Reply:
I have also lost track of this. Could you please provide a suggestion?

Reply:
My suggestion is to just revert to the original:

```suggestion
If you'd like to contribute to translations, you can visit <https://lukwebsforge.github.io/tldri18n/>
```